src/engraving/tests/parts_data/part-all-appendmeasures.mscx

```diff
@@ -1267,7 +1267,7 @@
                 <Fingering>
                   <linked>
                     </linked>
-                  <offset x="1.06662" y="-4.71271"/>
+                  <offset x="1.05809" y="-4.67501"/>
```

Comment:
Why did you change the test data?

Reply:
Ok I've double checked to be sure. In all of these tests, the spatium in the master score is slightly different than the spatium in the part, so all of these tests had slightly wrong offsets because of the wrong scaling calculations. Now all the new offset values should be correct.